core/src/main/java/nl/nn/adapterframework/configuration/IbisContext.java

```diff
@@ -166,8 +170,10 @@ public synchronized void close() {
 			classLoaderManager.shutdown();
 		}
 
-		log("shutdown in " + (System.currentTimeMillis() - start) + " ms"); //Should log this before the actual Context is destroyed
+		long shutdownTime = (System.currentTimeMillis() - start);
+		log("shutdown in " + shutdownTime + " ms"); //Should log this before the actual Context is destroyed
```

Comment:
Just like previous comment
```suggestion
		log("shutdown in {} ms", shutdownTime); //Should log this before the actual Context is destroyed
```